modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/UserDisplayContext.java

```diff
@@ -106,7 +106,8 @@ public Contact getContact() throws PortalException {
 		List<Group> groups = Collections.emptyList();
 
 		if (_selUser != null) {
-			groups = _selUser.getGroups();
+			groups = GroupLocalServiceUtil.getUserGroups(
+				_selUser.getUserId(), true);
```

Comment:
Do not include inherited sites here. We're using this method to show sites that the user is a direct member of in the user's site section. Including inherited sites here will cause duplicate entries to appear in the site section.

Reply:
Hi @pei-jung,

I'm afraid I don't understand this point. Currently, if we don't check for inherited sites, the site section disappears entirely if a user has an inherited site from an organization such as in the above reproduction steps. See the image below.

![sitegroups0](https://user-images.githubusercontent.com/1923745/52025657-0e10a680-24ba-11e9-9354-6878378faeb1.PNG)

What do you mean when you say that it will cause duplicate entries to appear?

Reply:
We should check for inherited sites but not in the `UserDisplayContext.getGroups()`. In edit user > general > membership, we have different tables for display user's `sites` and `inherited sites`. We use getGroups() and getInheritedSites() for that.